templates/cisco_ios_show_ip_mroute.template

```diff
@@ -9,7 +9,7 @@ Value REVERSE_PATH_FORWARDING_NEIGHBOUR_IP (\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})
 Value REGISTERING (Registering)
 Value List OUTGOING_INTERFACE (\S+)
 Value List FORWARD_MODE (Forward\/Sparse|Forward\/Dense)
-Value List OUTGOING_MULTICAST_UP_TIME (\d{2}:\d{2}:\d{2})
+Value List OUTGOING_MULTICAST_UP_TIME (\S+)
 Value List OUTGOING_MULTICAST_EXPIRATION_TIME (\d{2}:\d{2}:\d{2}|stopped)
```

Comment:
Shouldn't this line also be changed in that case?

Reply:
I agree. Do you mind addressing this?

Also, can you please add separate **.raw** and **.parsed** to the tests folder? This helps ensure backwards compatibility and more data to test against.